src/Components/Notes/NotesControl.test.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable require-await */
```

Comment:
you sure to disable this?  never seen this in our tests

Reply:
yes if this not disabled our lint complains, it is dealing with the way act is assembled in the test.

Reply:
yes, it fixes the issue with act and prevents lint from complaining.